benchmarks/build.gradle

```diff
@@ -43,6 +43,11 @@ task removeRedundantFiles(type: Delete) {
     delete "$buildDir/classes/kotlin/jmh/benchmarks/flow/misc/Numbers\$\$special\$\$inlined\$filter\$1\$1.class"
 }
 
+task runMonteCarloBasedChannelBenchmark(type : JavaExec) {
```

Comment:
This is a producer-consumer channel benchmark, which uses the Monte-Carlo method